### ISSUE TITLE
Cannot bake materials because of missing LBS key

### ISSUE THREAD

**Issue by FrankVargas427** [FlailingFog/KK-Blender-Porter-Pack#446]
As the title says, if I try to bake the material templates, regardless if it's dark, light or both, Blender throws this error:

```
Traceback (most recent call last):
  File "C:\Users\Frank\AppData\Roaming\Blender Foundation\Blender\3.5\scripts\addons\KK-Blender-Porter-Pack-650\exporting\bakematerials.py", line 422, in execute
    if mat.node_tree.nodes['Rim'].node_tree == bpy.data.node_groups['LBS']:
KeyError: 'bpy_prop_collection[key]: key "LBS" not found'
```

Doing some googling I came to find that this is related to the Lightning Boy Shader plugin. Thing is, I never installed it. I don't even own it. So I'm not 100% sure as to why this error is appearing if at no point during the process of this project have I used LBS. I didn't even know about it until this error.

![image](https://github.com/FlailingFog/KK-Blender-Porter-Pack/assets/43286109/ce16b4d6-6992-4759-a7dd-d14bc878cf70)

Commenting that whole entire section in the code, at least for this project, does seem to work, and no texture was broken after baking.

![Koikatu_M_20220915172321554_Black Key_key](https://github.com/FlailingFog/KK-Blender-Porter-Pack/assets/43286109/1cea8ca3-810d-4726-8d40-2099e9803f2b)

**Comment by HarryBols** [comment]
Switching to EEVEE for material baking...

Error:          Unknown python error occurred
Traceback (most recent call last):
  File "F:\Blender\Scripts\addons\KK-Blender-Scripts-master\exporting\bakematerials.py", line 422, in execute
    if mat.node_tree.nodes['Rim'].node_tree == bpy.data.node_groups['LBS']:
KeyError: 'bpy_prop_collection[key]: key "LBS" not found'

same issue

**Comment by FlailingFog** [comment]
Sorry, I made it such that you would be able to bake if you imported using LBS, but I guess I didn't realize that it would break if you bake **_without_** LBS.
Can you try [this latest master branch version](https://github.com/FlailingFog/KK-Blender-Porter-Pack/archive/refs/heads/master.zip) with a fix applied?

**Comment by HarryBols** [comment]
> Sorry, I made it such that you would be able to bake if you imported using LBS, but I guess I didn't realize that it would break if you bake **_without_** LBS. Can you try [this fixed master branch version](https://github.com/FlailingFog/KK-Blender-Porter-Pack/archive/refs/heads/master.zip)?

it baked but every single texture was white

**Comment by HarryBols** [comment]
even going back to 3.4 which supports lbs and re exporting the character looks fine but everything still bakes white

**Comment by FlailingFog** [comment]
this should be fixed in 6.6.0
